components/inputs/README.md

```diff
@@ -111,6 +112,30 @@ search.addEventListener('d2l-input-search-searched', (e) => {
 
 When the input is cleared, the same event will be fired with an empty value.
 
+## Select Lists
+
+Native `<select>` elements can be styled by importing `input-select-styles.js` into your LitElement and applying the `d2l-input-select` CSS class.
+
+Note: in order for RTL to function correctly, make sure your component uses the `RtlMixin`.
+
+```javascript
+import { RtlMixin } from '@brightspace-ui/core/mixins/rtl-mixin.js';
+import { selectStyles } from '@brightspace-ui/core/components/inputs/input-select-styles.js';
+class MyElem extends RtlMixin(LitElement) {
+	static get styles() {
```

Comment:
Super nit: Some sample code in the readmes use spaces, some use tabs.  I find the tabs in the GitHub readmes is too much, and sometimes it causes things to wrap that shouldn't.

Reply:
Haha, the eternal debate! Agreed though -- fixed.